src/validation.cpp

```diff
@@ -3860,13 +3860,13 @@ void PruneBlockFilesManual(CChainState& active_chainstate, int nManualPruneHeigh
     }
 }
 
-void CChainState::LoadMempool(const ArgsManager& args)
+void CChainState::LoadMempool(FopenFn mockable_fopen_function)
 {
     if (!m_mempool) return;
-    if (!m_mempool->m_persistence_path.empty()) {
-        ::LoadMempool(*m_mempool, *this);
-    }
-    m_mempool->SetLoadTried(!ShutdownRequested());
+    auto simple_atmp = [&](const CTransactionRef& tx, int64_t accept_time) EXCLUSIVE_LOCKS_REQUIRED(::cs_main) {
```

Comment:
I guess `simple_atmp` in opposition to the more complex `AcceptPackage` / `AcceptMultipleTransactions` ? In that case `single_tx_atmp` might be more speaking, I don't know.

Reply:
No longer relevant as of 80b303ea1a065602cf3f84d45b04dce95bfb6952